sdk/jdbc/azure-identity-providers-core/src/main/java/com/azure/identity/providers/jdbc/implementation/token/AccessTokenResolverOptions.java

```diff
@@ -27,7 +36,26 @@ public AccessTokenResolverOptions(Properties properties) {
         this.claims = AuthProperty.CLAIMS.get(properties);
 
         String scopeProperty = AuthProperty.SCOPES.get(properties);
-        this.scopes = scopeProperty == null ?  new String[0] : scopeProperty.split(",");
+        if (scopeProperty == null) {
+            scopeProperty = getDefaultScope(properties);
+        }
+        this.scopes = scopeProperty.split(",");
+    }
+
+    private String getDefaultScope(Properties properties) {
+        String ossrdbmsScope = OSS_RDBMS_SCOPE_MAP.get(AzureAuthorityHosts.AZURE_PUBLIC_CLOUD);
+        String authorityHost = AuthProperty.AUTHORITY_HOST.get(properties);
+        if (AzureAuthorityHosts.AZURE_PUBLIC_CLOUD.startsWith(authorityHost)) {
+            ossrdbmsScope = OSS_RDBMS_SCOPE_MAP.get(AzureAuthorityHosts.AZURE_PUBLIC_CLOUD);
+        } else if (AzureAuthorityHosts.AZURE_CHINA.startsWith(authorityHost)) {
+            ossrdbmsScope = OSS_RDBMS_SCOPE_MAP.get(AzureAuthorityHosts.AZURE_CHINA);
+        } else if (AzureAuthorityHosts.AZURE_GERMANY.startsWith(authorityHost)) {
+            ossrdbmsScope = OSS_RDBMS_SCOPE_MAP.get(AzureAuthorityHosts.AZURE_GERMANY);
+        } else if (AzureAuthorityHosts.AZURE_GOVERNMENT.startsWith(authorityHost)) {
+            ossrdbmsScope = OSS_RDBMS_SCOPE_MAP.get(AzureAuthorityHosts.AZURE_GOVERNMENT);
+        }
+        LOGGER.info("Ossrdbms scope set to ", ossrdbmsScope);
```

Comment:
should this be an info or a debug level?

Reply:
I prefer it to be an info log.

Reply:
```suggestion
        LOGGER.info("Ossrdbms scope set to {}", ossrdbmsScope);
```